### DISCUSSION TITLE
Slugs

### DISCUSSION THREAD

**Discussion by bjlamarca** [netbox-community/netbox#17621] [General]
Excuse if this a newbie question.  I am doing some development, and I see most of the NetBox models have slugs, but don't understand the need.  Should I be including a slug field in all my models?

**Comment by zmaster7** [comment]
There were previous issues/discussions regarding slugs in #8113 and #12315, and their purpose (or lack thereof) as it relates to NetBox.

Short Answer: [It is functionality inherited from django.](https://docs.djangoproject.com/en/5.1/ref/models/fields/#slugfield) It is more or less optional/can be ignored, unless you have a reason to utilize it.

**Reply by bjlamarca** [reply]
Thank you for the short answer!
